tests/resources/scripts/create_db.py

```diff
@@ -7,7 +7,7 @@
 # Author: Kyle Lahnakoski (kyle@lahnakoski.com)
 #
 import sys
-
+sys.path.append("../alert_manager")
```

Comment:
Running from the root directory of the project gives an error "ImportError: No module named config
". So, for correcting this, I wrote this code.